app/controllers/users.js

```diff
@@ -110,5 +112,27 @@ module.exports = {
         })
       });
     });
+  },
+  albumPhotosList(req, res, next) {
+    const purchase = {
+      userId: req.user.id,
+      albumId: req.params.id
+    };
+    Purchase.findOne({ where: purchase }).then(p => {
+      if (!p) {
+        next(defaultError('Album has not been purchased.'));
+      } else {
+        axios
+          .get(`https://jsonplaceholder.typicode.com/photos?albumId=${p.albumId}`)
+          .then(function(response) {
+            res.status(200).json({
+              photos: response.data.map(photo => {
```

Comment:
Same as the previous PR, this logic could be move somewhere else like a serializer to keep the controller code cleaner.